MekHQ/src/mekhq/gui/MekLabTab.java

```diff
@@ -601,10 +601,10 @@ void setTechFaction(int techFaction) {
         private static final long serialVersionUID = 4348862352101110686L;
         
         private SmallCraft entity;
-        private megameklab.com.ui.Dropship.tabs.DropshipStructureTab structureTab;
+        private megameklab.com.ui.aerospace.DropshipStructureTab structureTab;
         private megameklab.com.ui.Aero.tabs.EquipmentTab equipmentTab;
         private megameklab.com.ui.Aero.tabs.BuildTab buildTab;
-        private megameklab.com.ui.Dropship.tabs.TransportTab transportTab;
+        private megameklab.com.ui.aerospace.TransportTab transportTab;
```

Comment:
`DropshipStructureTab` and `DropshipStructureTab` have been moved in MML master: this is so that this PR compiles against MML master.